docs/chefops.md

```diff
@@ -0,0 +1,132 @@
+Running chef scrips
+===================
+The base class `SushiChef` provides a lot of command line arguments that control
+the chef script's operation. It is expected that **every chef script will come
+with a README** that explains the desired command line arguments for the chef script.
+We call "chef ops" all this additional context necessary to run the chef.
```

Comment:
A little confused by this line- did you mean "'Chef ops' describes those who are responsible for running the chefs"?